stdlib/multiprocessing/managers.pyi

```diff
@@ -61,32 +61,59 @@ class ValueProxy(BaseProxy, Generic[_T]):
     if sys.version_info >= (3, 9):
         def __class_getitem__(cls, item: Any, /) -> GenericAlias: ...
 
-class DictProxy(BaseProxy, MutableMapping[_KT, _VT]):
-    __builtins__: ClassVar[dict[str, Any]]
-    def __len__(self) -> int: ...
-    def __getitem__(self, key: _KT, /) -> _VT: ...
-    def __setitem__(self, key: _KT, value: _VT, /) -> None: ...
-    def __delitem__(self, key: _KT, /) -> None: ...
-    def __iter__(self) -> Iterator[_KT]: ...
-    def copy(self) -> dict[_KT, _VT]: ...
-    @overload  # type: ignore[override]
-    def get(self, key: _KT, /) -> _VT | None: ...
-    @overload
-    def get(self, key: _KT, default: _VT, /) -> _VT: ...
-    @overload
-    def get(self, key: _KT, default: _T, /) -> _VT | _T: ...
-    @overload
-    def pop(self, key: _KT, /) -> _VT: ...
-    @overload
-    def pop(self, key: _KT, default: _VT, /) -> _VT: ...
-    @overload
-    def pop(self, key: _KT, default: _T, /) -> _VT | _T: ...
-    def keys(self) -> list[_KT]: ...  # type: ignore[override]
-    def items(self) -> list[tuple[_KT, _VT]]: ...  # type: ignore[override]
-    def values(self) -> list[_VT]: ...  # type: ignore[override]
-    if sys.version_info >= (3, 13):
+if sys.version_info >= (3, 13):
+    class _BaseDictProxy(BaseProxy, MutableMapping[_KT, _VT]):
+        __builtins__: ClassVar[dict[str, Any]]
+        def __len__(self) -> int: ...
+        def __getitem__(self, key: _KT, /) -> _VT: ...
+        def __setitem__(self, key: _KT, value: _VT, /) -> None: ...
+        def __delitem__(self, key: _KT, /) -> None: ...
+        def __iter__(self) -> Iterator[_KT]: ...
+        def copy(self) -> dict[_KT, _VT]: ...
+        @overload  # type: ignore[override]
+        def get(self, key: _KT, /) -> _VT | None: ...
+        @overload
+        def get(self, key: _KT, default: _VT, /) -> _VT: ...
+        @overload
+        def get(self, key: _KT, default: _T, /) -> _VT | _T: ...
+        @overload
+        def pop(self, key: _KT, /) -> _VT: ...
+        @overload
+        def pop(self, key: _KT, default: _VT, /) -> _VT: ...
+        @overload
+        def pop(self, key: _KT, default: _T, /) -> _VT | _T: ...
+        def keys(self) -> list[_KT]: ...  # type: ignore[override]
+        def items(self) -> list[tuple[_KT, _VT]]: ...  # type: ignore[override]
+        def values(self) -> list[_VT]: ...  # type: ignore[override]
+
+    class DictProxy(_BaseDictProxy[_KT, _VT]):
         def __class_getitem__(cls, args: Any, /) -> Any: ...
```

Comment:
While we're here — I assume this always returns instances of `GenericAlias`?

```suggestion
        def __class_getitem__(cls, args: Any, /) -> GenericAlias: ...
```

Reply:
I believe so.